model_compression_toolkit/core/pytorch/reader/graph_builders.py

```diff
@@ -178,11 +202,22 @@ def nodes_builder(model: GraphModule,
                                                  node.args[0].op == PLACEHOLDER and node.args[0].meta[TYPE] in (list, tuple))
             tensor_input_index = []
             op_call_args = list(node.args)
+            for arg_name, arg_val in _args_from_kwargs.items():
+                _args = get_node_args_spec(node)
+                if arg_name in _args:
+                    arg_index = _args.index(arg_name)
+                    # TODO: This code verifies the kwargs with Node values that are used as args are in the correct order.
```

Comment:
what's the "todo"? even if it just so it will be clear what's going on here, add a sentence about how it should be eventually handled